JavaFX static libs missing for Java project when target = android

Trying to build [HelloWorld](https://github.com/gluonhq/client-samples/tree/master/Maven/HelloWorld) for Android. After the linking is done, the build complains about not able to find **JavaFX static libs**. Given HelloWorld is a Java project with no dependency on JavaFX, this seems like a bug.

```
[INFO] --- client-maven-plugin:0.1.10-SNAPSHOT:link (default-cli) @ helloworld ---
started linking
done linking
java.nio.file.NoSuchFileException: ~/.gluon/substrate/javafxStaticSdk/14-ea+gvm2/android-aarch64/sdk/lib/libfreetype.so
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixCopyFile.copy(UnixCopyFile.java:548)
	at java.base/sun.nio.fs.UnixFileSystemProvider.copy(UnixFileSystemProvider.java:258)
	at java.base/java.nio.file.Files.copy(Files.java:1294)
	at com.gluonhq.substrate.target.AndroidTargetConfiguration.link(AndroidTargetConfiguration.java:186)
	at com.gluonhq.substrate.SubstrateDispatcher.nativeLink(SubstrateDispatcher.java:227)
	at com.gluonhq.NativeLinkMojo.execute(NativeLinkMojo.java:54)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:309)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:194)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:107)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:955)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:290)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:194)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:31 min
[INFO] Finished at: 2019-12-26T15:45:44+05:30
[INFO] Final Memory: 17M/64M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.gluonhq:client-maven-plugin:0.1.10-SNAPSHOT:link (default-cli) on project helloworld: Error: ~/.gluon/substrate/javafxStaticSdk/14-ea+gvm2/android-aarch64/sdk/lib/libfreetype.so -> [Help 1]
```

--- a/src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java
+++ b/src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java
@@ -189,15 +189,22 @@ public class AndroidTargetConfiguration extends PosixTargetConfiguration {
         Path graalLibPath = dalvikLibArm64Path.resolve("libmygraal.so");
         Files.deleteIfExists(graalLibPath);
         Files.copy(libPath, graalLibPath);
-        Path freetypeLibPath = dalvikLibArm64Path.resolve("libfreetype.so");
-        Files.deleteIfExists(freetypeLibPath);
-        Files.copy(fileDeps.getJavaFXSDKLibsPath().resolve("libfreetype.so"), freetypeLibPath);
 
-        ProcessRunner aaptAddLibs = new ProcessRunner(aaptCmd, "add", unalignedApk,
-                "lib/arm64-v8a/libmygraal.so","lib/arm64-v8a/libfreetype.so" );
+        boolean useJavaFX = projectConfiguration.isUseJavaFX();
+        if (useJavaFX) {
+            Path freetypeLibPath = dalvikLibArm64Path.resolve("libfreetype.so");
+            Files.deleteIfExists(freetypeLibPath);
+            Files.copy(fileDeps.getJavaFXSDKLibsPath().resolve("libfreetype.so"), freetypeLibPath);
+        }
+
+        List<String> aaptAddLibsArgs = Arrays.asList(aaptCmd, "add", unalignedApk,"lib/arm64-v8a/libmygraal.so");
+        if (useJavaFX) {
+            aaptAddLibsArgs.add("lib/arm64-v8a/libfreetype.so");
+        }
+
+        ProcessRunner aaptAddLibs = new ProcessRunner(aaptAddLibsArgs.toArray(String[]::new));
         processResult = aaptAddLibs.runProcess("AAPT-add lib", dalvikPath.toFile());
-        if (processResult != 0)
-            return false;
+        if (processResult != 0) return false;
 
         ProcessRunner zipAlign = new ProcessRunner(buildToolsPath.resolve("zipalign").toString(), "-f", "4", unalignedApk, alignedApk);
         processResult = zipAlign.runProcess("zipalign");

